src/dev-app/cdk-experimental-menu/cdk-menu-demo.html

```diff
@@ -0,0 +1,23 @@
+<ul cdkMenuBar class="example-menu-bar">
+  <li role="none"><button id="file_button" [cdkMenuTriggerFor]="file">File</button></li>
```

Comment:
Not in this PR, but at some point I want to sit down and think about the way we're using lists with the menu and see if we could do something simpler